simulation/README.md

```diff
@@ -5,64 +5,44 @@ Additionally, it publishes a [TwistStamped](http://docs.ros.org/melodic/api/geom
 
-#### Downloading on Mac
-1. Download the mac version of CoppeliaSim Edu from [this link](http://coppeliarobotics.com/downloads)
-2. Unzip the download and move the resulting folder to Applications
-3. To run, click on coppeliaSim inside of that folder
-4. If you get an error about models not found, run the following at the top of the CoppeliaSim folder
+#### Installing on Mac
+1. Unzip the download and move the resulting folder to Applications
+2. To run, click on CoppeliaSim inside of that folder
+3. If you get an error about models not found, run the following at the top of the CoppeliaSim folder
 ```bash
 sudo xattr -r -d com.apple.quarantine *
 ```
 
 ### Downloading Remaining Files
 You'll be using files in `robosub-ros/simulation/personal`.
 
-Make sure you have some version of python installed on your personal computer. **If you have a Mac, you must have Python 3!** You can download and install it at [this link](https://www.python.org/downloads/release/python-381/).
-
-<details>
-    <summary>Click for Explanation of Python3</summary>
-    You will need to run a python script on your personal computer (either from command line or from IDE) to have the simulations communicate with each other. For Macs, this script uses a library, which can cause permission errors when run by restricted programs. Because Python 2 is deprecated, it is a restricted program, while Python 3 is not.
-</details>
+Make sure you have some version of Python3 installed on your personal computer. You can download and install it at [this link](https://www.python.org/downloads/release/python-381/).
 
 ## Running the Simulation
 ### Docker Simulation Setup
-1. Open up a terminal and ssh into your Docker Container with the `-XY` flag (i.e. run `ssh -XY -p 2200 duke@[ip address]`).
-2. Run `source /opt/ros/kinetic/setup.bash` and `roscore &`.
-3. Run `cd dev/robosub-ros/simulation` and then `./runSim.sh &` (you may first need to run `chmod +x runSim.sh`). Wait until the terminal says `Initialization successful.` If it delays on the video compression library or meshcalc for an extended period of time, press enter a couple of times. This may be nothing more than confirmation-bias superstition.
-4. Run whatever ROS topic publishing code you have. In `robosub-ros/simulation/docker`, there is a python script `squareCommand.py` that you can run to make the robot move approximately in a square. (In reality, the robot will spin in wide circles because the robot isn't balanced.) You can use this script to test if communication between the simulations is working.
+1. In the landside container, make sure `roscore` is running.
+2. Run `cd dev/robosub-ros/simulation` and then `./runSim.sh &` (you may first need to run `chmod +x runSim.sh`). Wait until the terminal says `Initialization successful.` If it delays on the video compression library or meshcalc for an extended period of time, press enter a couple of times. This may be nothing more than confirmation-bias superstition.
```

Comment:
The problem with this is that roscore should only be run in the onboard container. By doing the
 `source /opt/ros/melodic/setup_network.bash` you are saying that the roscore is running in the onboard container. I think you can do something like:
1. In the onboard container, make sure `roscore` is running.
2. Now, in the landside container, run ...